lib/cmsis_rtos_v1/Kconfig

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+config CMSIS_RTOS_V1_IPC
```

Comment:
What is the IPC for here? You probably copied that from the POSIX API which is also wrong. I would remove that and just call it CMSIS_RTOS_V1

Reply:
Yes. Had copied from POSIX. Will remove.